docs/contributing.md

```diff
@@ -63,3 +66,33 @@ Additionally, we strive to use the ["Google" docstring format](https://sphinxcon
 If you have `clang-format` installed, you can format the proto files by running `clang-format --style=file -i *.proto` in `//app/proto`.
 
 In the **web frontend**, make sure to run `yarn format` in `//app/web` before asking for a review.
+
+## Project Management
+
+Couchers contributions are managed by all teams collectively within the [GitHub Project](https://github.com/orgs/Couchers-org/projects/4). Product development is done in pursuit of quarterly release milestones by which the team strives to release platform features and content relevant to our primary objectives. Consistency and focus is maintained through Scrum-style development cycles every month during which we develop, assess, plan, and repeat both asynchronously and in weekly discussion meetings.
```

Comment:
> Couchers contributions are managed by all teams collectively

Teams other than dev have not historically used github. Though I would like that to change going forward, I think we can accept being a "software non-profit" that coordinates here.

Re: the next sentence. I think working around quarterly release milestones (aka a blog post with some updates and using that as a deadline) might be a good way to go. I'm not sure if "Scrum-style" is accurate, maybe we should call it just "Scrum-inspired", though I've never been in a team where "Scrum" really meant so much.

Let's discuss this in the upcoming meeting!